Assets/MRTK/Providers/Experimental/WindowsMixedReality/SceneUnderstanding/.gitignore

```diff
@@ -1,5 +0,0 @@
-# ========================= #
-# MSBuildForUnity generated #
-# ========================= #
-.obj
-.bin
```

Comment:
Just want to make sure we really wish to remove these as they seem reasonable

Reply:
i did not intend to remove those

Reply:
i will add them to the repo's .gitignore file